### PR TITLE
#166790007 Pagination Support for Articles

### PR DESCRIPTION
**Pull request by Hervera**
#### What does this PR do?
It provides pagination support for articles

#### Description of Task to be completed
- add pagination of 10 ratings per page
- add page and/or limit as query params
- add metadata on every article's ratings page

#### How should this be manually tested?
1. Run the following commands in the terminal
```
git clone https://github.com/andela/ah-92explorers-backend.git
cd ah-92explorers-backend
git checkout ft-pagination-for-articles-166790007
npm install
npm start
```
2. Create a .env file and add the following snippets
```
DATABASE_URL=postgresql://dbusername:dbpassword@localhost:5432/dbname
DATABASE_TEST=postgresql://dbusername:dbpassword@localhost:5432/dbnamefortest
SECRET=whatever you want it to be
```
3. Install `sequelize` globally
```
sudo npm install -g sequelize
```
4.  Create a database using `sequelize db:create`
5. Install and setup Postman to consume `pagination for articles` endpoint
6. Run the following requests in Postman using `localhost:3000`
## Endpoint
|  Method  |  Endpoint  |  Task  |
|  --- |  --- |  ---  |
|  `GET`  |  `/api/articles/feed?page=1&limit=1`  |  `pagination support for articles`  |

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#166790007](https://www.pivotaltracker.com/story/show/166790007)

#### Screenshots (if appropriate)
Screen Shot 2019-07-23 at 17 10 40

#### Checklist

- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I don't have more than 3 major commits on this PR

### CODE REVIEW

File: controllers/ratings.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
+import models from '../models';
```
Comment (niyoceles) [inline controllers/ratings.js]: Remove this file `ratings.js` in controllers will no longer be used.

**Review by niyoceles — CHANGES_REQUESTED** [review]
try to remove the files which are related to rating an article, some of those files have been changed in this `ft-user-rating-article-166789999` branch

**Review by elemanhillary-zz — DISMISSED** [review]
ready for merge

**Review by niyoceles — APPROVED** [review]

**Review by kodek-sleuth — APPROVED** [review]

**Review by elemanhillary-zz — APPROVED** [review]
ready for merge

**Pull request merged**